### DISCUSSION TITLE
does this support a specific page size?

### DISCUSSION THREAD

**Discussion by jeffery9** [docker-library/postgres#1178] [General]

**Comment by yosifkit** [comment]
> Every table and index is stored as an array of pages of a fixed size (usually 8 kB, although a different page size can be selected when compiling the server).
>
>\- https://www.postgresql.org/docs/current/storage-page-layout.html

Looking at this bit of PostgreSQL documentation, I don't think so.

For the Debian based images, we just use the `apt` package repos provided on `http://apt.postgresql.org/pub/repos/apt/`. For the Alpine based images, we had mirrored the config used to build the Debian packages.
